### ISSUE TITLE
Available options/default value for duplicateProperty

### ISSUE THREAD

**Issue by jwilsson** [lesshint/lesshint#14]
Do we need any more options? Are we satisfied with the current default value? Any other thoughts?

Current options:
None, do we need any options or is it enough with enabled: true/false?

Please cast a vote for one of the options or post your other thoughts or suggestions for options.

**Comment by kokarn** [comment]
Disable should really be an option, it's very useful for things like fonts
```
@font-face {
    font-family: 'Volvo Sans';
    src: url("fonts/my-font.eot");
    src: url("fonts/my-font.eot?#iefix") format('embedded-opentype'),
        url("fonts/my-font.woff2") format('woff2'),
        url("fonts/my-font.woff") format('woff'),
        url("fonts/my-font.ttf") format('truetype');
    font-weight: 500;
    font-style: normal;
}
```

**Comment by jwilsson** [comment]
You mean like an `exclude` option?

For example:

    {
        "duplicateProperty": {
            "enabled": true,
            "exclude": ["src"]
        }
    }

**Comment by kokarn** [comment]
Maybe, i'm not sure.
There is a few other valid ways to duplicate properties, linear gradient background fallbacks etc.

**Comment by jwilsson** [comment]
I have an idea about enabling/disabling linters on a specific row or a specific file with inline comments, something like this:

    // lesshint:disable duplicateProperty

or

    // lesshint duplicateProperty: false

Or some other way, it's just an idea at this time and I have some implementation details to figure out. How does this sound?

**Comment by kokarn** [comment]
It sounds like it could be really useful. 
I'd go with the second version of the syntax i think. 
Dunno what will be better for the future tho.

**Comment by kokarn** [comment]
The exclude parameter is implemented and the inline version is tracked in #55. Closing
